extensions/package.json

```diff
@@ -10,13 +10,13 @@
     "postinstall": "node ./postinstall.mjs"
   },
   "devDependencies": {
-    "@parcel/watcher": "2.1.0",
+    "@parcel/watcher": "2.5.0",
     "esbuild": "0.23.0",
     "vscode-grammar-updater": "^1.1.0"
   },
   "overrides": {
     "node-gyp-build": "4.8.1",
-    "@parcel/watcher@2.1.0": {
+    "@parcel/watcher@2.5.0": {
```

Comment:
This override can be removed, 2.5.0 watcher version comes with the required node-addon-api.